src/expressions/if-expr.md

```diff
@@ -1,14 +1,22 @@
-# `if` and `if let` expressions
+# `if` expressions
 
-## `if` expressions
+## Syntax
+
+The same syntax is used by `if`, `if let` and chains of expressions.
 
 > **<sup>Syntax</sup>**\
 > _IfExpression_ :\
-> &nbsp;&nbsp; `if` [_Expression_]<sub>_except struct expression_</sub> [_BlockExpression_]\
-> &nbsp;&nbsp; (`else` (
->   [_BlockExpression_]
-> | _IfExpression_
-> | _IfLetExpression_ ) )<sup>\?</sup>
+> &nbsp;&nbsp; `if` _IfLetList_ [_BlockExpression_]\
+> &nbsp;&nbsp; ( `else` _IfLetList_ [_BlockExpression_] )<sup>\?</sup>
```

Comment:
This grammar doesn't look correct to me. The `else` part seems to have multiple issues:

* Doesn't allow `if foo {} else {}` where it is an unconditional else block.
* Doesn't allow `if foo {} else if bar {}` where there is an `else if`.
* Seems to allow `if foo {} else bar {}` which isn't valid.